src/main/java/com/develhope/spring/model/user/Rental.java

```diff
@@ -0,0 +1,47 @@
+package com.develhope.spring.model.user;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public class Rental {
+    private LocalDate startDate;
+    private LocalDate endDate;
+    private BigDecimal dailyCost;
+    private BigDecimal totalCost;
+    private boolean paid;
+
+    //it was decided to use vehicle (object) but instead of using
+    // Vehicle as object is created private int vehicleId;
+    // passed to the constructor
+    // handled the same way in the Order class of User
+    private int vehicleId;
+
+
+    public Rental(LocalDate startDate, LocalDate endDate, BigDecimal dailyCost, int vehicleId) {
+        this.startDate = startDate;
+        this.endDate = endDate;
+        this.dailyCost = dailyCost;
+        this.vehicleId = vehicleId;
+        this.calculateTotalCost();
+        this.paid = false;
+    }
+
+    private void calculateTotalCost() {
+        long rentalDays = startDate.until(endDate).getDays();
+        this.totalCost = dailyCost.multiply(BigDecimal.valueOf(rentalDays));
+    }
+
+    public void markAsPaid() {
+        this.paid = true;
+    }
+
+    // Getters
+    public BigDecimal getTotalCost() {
+        return totalCost;
+    }
+
+    public boolean isPaid() {
```

Comment:
Come dissi ad alessio questo è un semplice get. chiamalo getPaid

Reply:
comunque è un boolean, lo stesso IntelliJ lo suggerisce così, un po' come dire "Il veicolo è pagato: vero"

Reply:
normalmente se usi il generatore dell'IDE ti chiama con get tutti i metodi che ritornano una variabile d'istanza mi pare. Il ragionamento di definirlo isPaid è corretto naturalmente, però in questo caso è da vedere quale sia la convenzione

Reply:
é proprio l'IDE che il getter dei booleani te lo dà con "is"

Reply:
apposto allora come detto in chiamata deve essere un nuovo aggiornamento

Reply:
non seguo il ragionamento. 

Però vorrei farvi notare che prima avendo inizializzato a false 

con il metodo di sopra vediamo lo stato ( in questo senso ti seguo se lo definisce getter)

poi abbiamo il metodo

 public void markAsPaid() {
        this.paid = true;
    }
    
per renderlo da false a true

Per favore, siate più espliciti nella richiesta di modifica